pkg/client/ocm/client.go

```diff
@@ -546,15 +546,21 @@ func (c client) GetQuotaCosts(organizationID string, fetchRelatedResources, fetc
 
 	var quotaCosts []*amsv1.QuotaCost
 
+	// iterates through all Quota and only return the quota whose resources matches the given filters
 	quotaCostList.Each(func(qc *amsv1.QuotaCost) bool {
 		relatedResources := qc.RelatedResources()
+		quotaMatchesFilters := false
 		for _, relatedResource := range relatedResources {
 			for _, filter := range relatedResourceFilters {
 				if filter.IsMatch(relatedResource) {
-					quotaCosts = append(quotaCosts, qc)
-					return false
+					quotaMatchesFilters = true
```

Comment:
Is this correct? Shouldn't it match **all** filters?

Reply:
After checking the code it seems it should add it if one of the filters match so the current code is correct.

Can we document this behavior around filters on this method documentation to avoid future confusions?

Reply:
cc @JameelB to confirm